portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,45 +29,38 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet responsible for creating new comments and listing the comments. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Prefer to make this member `final`, as you don't intend to change it throughout the program.